HACKING.md

```diff
@@ -322,8 +329,8 @@ A local cache is maintained in `~/.cache/cockpit-dev`.
 
 ## Contributing a change
 
-Make a pull request on github.com with your change. All changes get reviewed,
-tested and iterated on before getting into Cockpit. The general workflow is
+Make a pull request on GitHub.com with your change. All changes get reviewed,
```

Comment:
Not a strong opinion, but capitalizing domain names is rather unusual. (Feel free to ignore)